src/app/pages/flex-page/blocks/RichTextBlock.scss

```diff
@@ -11,17 +11,8 @@
     margin-top: 0;
   }
 
-  h1 {
-    @include title-font(4.8rem);
-  }
-  h2 {
-    @include title-font(2.6rem);
-  }
-  h3 {
-    @include title-font(1.8rem);
-  }
-  h4, h5, h6 {
-    @include title-font(1.6rem);
+  h5, h6 {
+    @extend %text-h4;
```

Comment:
I had forgotten about the `%text-` stuff! Same as `@include set-font(h4)` if you prefer `@include`.

Reply:
i copied this out of the pattern library sass, i don't really care either way